### PR TITLE
Appends FRQ so as not to wipe out other arguments if they exist

### PR DESCRIPTION
**Pull request by jontours**
Other form types were being erased by the logic, this preserves them adds FRQ.

### CODE REVIEW

File: webservices/resources/filings.py [hunk 1]
```diff
@@ -61,7 +61,10 @@ def args(self):
 
     def get(self, **kwargs):
         if kwargs.get('form_type') and 'RFAI' in kwargs.get('form_type'):
-            kwargs['form_type'] = ['RFAI', 'FRQ']
+            #If user happened to also pass in 'FRQ', remove it to prevent unnecessary filtering
+            #by appending to the list we don't wipe out the other form_type(s) passed in
+            kwargs.get('form_type').remove('FRQ')
```
Comment (LindsayYoung) [inline webservices/resources/filings.py]: Seems like we should leave this if that is what someone is explicitly looking for

Comment (jontours) [inline webservices/resources/filings.py]: If somebody is explicitly looking for `FRQ` files this will not alter that behavior.

Comment (LindsayYoung) [inline webservices/resources/filings.py]: OK, thanks for explaining!

Comment (jontours) [inline webservices/resources/filings.py]: It's much simpler to just remove it, whatever performance hit on database is minimal (if any at all) is acceptable if we keep the logic simple.

**Pull request merged**